projects/onto-search/src/lib/onto-search-column-selector/onto-search-column-selector.component.scss

```diff
@@ -0,0 +1,23 @@
+.column-selector-component {
```

Comment:
Is this selector correct? I couldn't find where it's used. Did you mean `onto-column-selector-component`?

Reply:
yes i ment `onto-column-selector-component`